common/persistence/cassandraPersistence.go

```diff
@@ -2035,6 +2110,14 @@ func isTimeoutError(err error) bool {
 	return ok
 }
 
+func isThrottlingError(err error) bool {
```

Comment:
What about other persistence calls?  We should also wrap them within ServerBusyEasy.